### ISSUE TITLE
Warn on import value only used in value context

### ISSUE THREAD

**Issue by arv** [gajus/eslint-plugin-flowtype#193]
I'm not sure this belongs here or in facebook/flowtype?

I want the following to be considered an error.

```js
// c.js
export default class C {}

// i.js
import C from './c.js';  // error C is only used in type context, use import type instead
let c: C;
...
```

I want value imports that are only used in type context to be flagged as errors.

I know Flow does the oppositie; types used on value context is an error.

**Comment by gajus** [comment]
Closing as stale.
